### ISSUE TITLE
Get help button

### ISSUE THREAD

**Issue by Nekiomi** [grummble-ai/PathOfExile-SkillGemMargins#5]
For some reason when I'm clicking "Get help" Button it doesn't send me any invite. Just opens discord in my browser. I wanted to ask if I can somehow put not a Minimum Buy-In threshold but maximum. Because I don't have infinite budget and obviously it is more profitable for me to lvlup 10 low price gems rather than 1 but expensive one in cases when margin is nearly the same.

**Comment by grummble-ai** [comment]
The link directs you to the corresponding channel in my discord (can you think of a better way to give feedback?). No worries though, I've already added what you asked for. Your feedback is very much appreciated!

**Comment by grummble-ai** [comment]
Done

**Comment by Nekiomi** [comment]
Since I'm not on your server it just shows me this.
![image](https://user-images.githubusercontent.com/17802270/217435812-611e9493-45ed-44d7-be2f-c0be8f56d41f.png)
I now figured it out that I had to join server before clicking the button but It is a bit confusing. And ether new feature is not working or I use it wrong. 
![image](https://user-images.githubusercontent.com/17802270/217436505-3816cf50-806f-4cfe-96f6-9e2ac39783be.png)
The first 2 gems shouldn't be here, no?
